blocks/lead-art-block/features/leadart/default.jsx

```diff
@@ -261,6 +261,25 @@ export const LeadArtPresentation = (props) => {
 
 	const lead_art = content?.promo_items?.lead_art || content?.promo_items?.basic || {};
 	const leadArtContent = getLeadArtContent(lead_art);
+
+	useEffect(() => {
+		if(leadArtContent?.type === "image"){
+			if (document.fullscreenEnabled) {
+				document.addEventListener("fullscreenchange", () => {
+					if (!document.fullscreenElement) {
+						setIsOpen(false);
+					}
+				});
+			} else if (document.webkitFullscreenEnabled) {
+				document.addEventListener("webkitfullscreenchange", () => {
+					if (!document.webkitFullscreenElement) {
+						setIsOpen(false);
+					}
+				})
+			}
+		}
+	}, [leadArtContent]);
+
```

Comment:
This whole block should be towards the top of this component, right after the variables are pulled out of customFields. React hooks need to be before any code that can cause a return or, as in this case, code that could throw an error (the try/catch for the adblock).

Reply:
I moved it here because useEffect is depending on `lead_art,` `leadArtContent` and `getLeadArtContent`. So I moved it after they defined.

Reply:
@vgalatro Moved UseEffect to top